bindata/network/multus-admission-controller/monitor.yaml

```diff
@@ -26,8 +26,13 @@ spec:
     tlsConfig:
       ca:
         configMap:
+{{- if and (.HyperShiftEnabled) (ne .CAConfigMap "")}}
```

Comment:
I thought the `CAConfigMap` check is not needed?

Reply:
Fixed